hoodie-client/src/main/java/com/uber/hoodie/io/HoodieMergeHandle.java

```diff
@@ -230,6 +234,14 @@ public Path getOldFilePath() {
     return oldFilePath;
   }
 
+  private Path getNewFilePath() {
```

Comment:
why is this named differently than in createHandle? can this be a single line ternary operator in both cases to save some LOC?